tasks/Module 2/task_1.py

```diff
@@ -0,0 +1,81 @@
+"""Task 1 and 2 of Module 2."""
+
+from collections import defaultdict
+# create CONSTANT variables for weight
+LIGHT_WEIGHT_THRESHOLD = 60
+HEAVY_WEIGHT_THRESHOLD = 70
+MAX_INVENTORY_CAPACITY = 80
+
+stuff = {'rope': 1, 'torch': 6, 'gold coin': 42, 'dagger': 1, 'arrow': 12}
+# update dragonLoot variable changed to dragon_loot
+dragon_loot = [
+
+    'gold coin', 'chewed gum', 'dagger', 'gold coin', 'gold coin', 'ruby',
+
+    'rubbish', 'chewed gum', 'used tissue']
+
+
+def display_inventory(inventory):
+    # change header for display_inventory function
+    """Check the inventory of a player and provides weight/speed dependency status."""
+    print("Inventory:")
+    item_total = 0
+
+    for item_name, item_count in inventory.items():
+        # change print iteams , to use default print() separator settings
+        print(item_name, item_count)
+        # change item_total counter to use augmented assignment
+        item_total += 1
+
+    print("Total number of items: " + str(item_total))
+    # here notify our hero if he/she is carrying too much
+    if item_total >= LIGHT_WEIGHT_THRESHOLD:
```

Comment:
I think the logic here should be the other way around. We should start with the heaviest and go to the lowest weight. If `item_total` will exceed `MAX_INVENTORY_CAPACITY`, then this statement will be fired, as opposed to statement from line 40.